test/unit/proxy_statuses/proxy_status_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class ProxyStatusTest < ActiveSupport::TestCase
+  setup do
+    @proxy = FactoryGirl.
+      build_stubbed(:template_smart_proxy, :url => 'https://secure.proxy:4568')
+  end
+
+  test '#versions_cache_key' do
+    proxy_status = ProxyStatus::Version.new(@proxy)
+    assert_equal("proxy_#{@proxy.id}/versions", proxy_status.cache_key)
```

Comment:
The cache key should be 'proxy_id/Version' - http://ci.theforeman.org/job/test_develop_pr_core/7801/database=mysql,label=fast,ruby=2.2/testReport/junit/%28root%29/ProxyStatusTest/test_0001__versions_cache_key/